apps/docs-snippets/src/guide/introduction/getting-started.test.ts

```diff
@@ -0,0 +1,43 @@
+import { FUEL_NETWORK_URL, Provider, Signer, Wallet } from 'fuels';
+
+/**
+ * @group node
+ * @group browser
+ */
+describe('Getting started', () => {
+  beforeAll(() => {
+    // Avoids using the actual network.
+    vi.spyOn(Provider, 'create')
+      .mockImplementationOnce(
+        () => Provider.create(FUEL_NETWORK_URL)
+      )
+  })
+
+  it('can connect to a local network', async () => {
+    // #region connecting-to-the-local-node
+    // #import { Provider, Wallet, Signer, FUEL_NETWORK_URL };
+
```

Comment:
I think it might be worth having `launchNode` in this example, what do people think?

Reply:
I'd say just link to it rather than duplicate examples

Reply:
I'd agree for any other page - however, for a getting started page - I'd say there is merit in having an all-in-one snippet (for the quickest route to get started).

Reply:
Will leave it for now.